components/kyma-environment-broker/internal/broker/instance_get_test.go

```diff
@@ -90,6 +90,42 @@ func TestGetEndpoint_GetProvisioningInstance(t *testing.T) {
 	assert.Len(t, response.Metadata.Labels, 2)
 }
 
+func TestGetEndpoint_GetInstanceWhereDeletedAtIsNotZero(t *testing.T) {
```

Comment:
In this test we test NOT GETTING the instance.

Reply:
My bad. I will rename it.